### DISCUSSION TITLE
record 클래스에 대해 설명해주세요.

### DISCUSSION THREAD

**Discussion by gyungchan-jo** [2024-woowacourse-study/level-interview#102] [Java]
.

**Comment by 3Juhwan** [comment]
record 클래스는 Java 14부터 도입된 불변 데이터를 표현하기 위한 간단한 버전의 클래스입니다. 필드는 final로 선언되어 값을 변경할 수 없고, equals(), hashCode(), toString(), getter()가 자동으로 생성됩니다. 주로 DTO나 불변성이 필요한 경우에 사용합니다.

**Comment by J-I-H-O** [comment]
record는 불변 데이터를 간단하게 표현하기 위한 새로운 형태의 클래스입니다.
필드는 final로 선언되어 불변성을 가집니다. 또한 필드에 대한 getter, equals(), hashCode(), toString() 메서드가 자동으로 생성되어 불필요한 코드 작성을 줄일 수 있습니다.
DTO나 VO의 용도로 사용할 수 있습니다.
